tests/e2e/utils/data_builder/common.py

```diff
@@ -0,0 +1,9 @@
+import secrets
```

Comment:
Nice! I usually use random for building random value

Reply:
Yeah, secrets solves the rng problem and chars that might not be allowed in CFN resources (token_urlsafe)